src/minimega/meshage.go

```diff
@@ -168,8 +163,15 @@ func meshageSend(c *minicli.Command, hosts string) (chan minicli.Responses, erro
 
 		log.Debug("meshage sent, waiting on %d responses", len(recipients))
 
+		// host -> response
 		resps := map[string]*minicli.Response{}
 
+		timeout := meshageTimeout
+		// If the timeout is 0, set to "unlimited"
+		if timeout == 0 {
+			timeout = math.MaxInt64
```

Comment:
Hah. Fortunately, time.Duration is a nanosecond count so that's only ~290 years.